manifests/j/JohnMacFarlane/Pandoc/3.1.13/JohnMacFarlane.Pandoc.installer.yaml

```diff
@@ -0,0 +1,31 @@
+# Created with komac v2.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: JohnMacFarlane.Pandoc
+PackageVersion: 3.1.13
+InstallerLocale: en-US
+InstallerType: wix
+Scope: user
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: uninstallPrevious
+ProductCode: '{E49FF226-1C35-4DAC-9652-78E4863CBAAA}'
+ReleaseDate: 2024-04-07
+AppsAndFeaturesEntries:
+- DisplayName: Pandoc 3.1.13
+  UpgradeCode: '{A68E8EF6-ABB1-4F22-A3C5-68DFDF0AB562}'
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/jgm/pandoc/releases/download/3.1.13/pandoc-3.1.13-windows-x86_64.msi
+  InstallerSha256: 51DA43AAEB93926921BEB777D18B2362D9B6A785D1333177291BB5E8834A4874
+  InstallerSwitches:
+    Custom: ALLUSERS="2"
+- Architecture: x64
+  InstallerUrl: https://github.com/jgm/pandoc/releases/download/3.1.13/pandoc-3.1.13-windows-x86_64.msi
+  InstallerSha256: 51DA43AAEB93926921BEB777D18B2362D9B6A785D1333177291BB5E8834A4874
+  InstallerSwitches:
+    Custom: ALLUSERS=1

```

Comment:
Should one of these have the `user` scope from above, and the other have `machine` scope?

Reply:
@russellbanks, the scope is missing 😭 .